### PR TITLE
[1LP][RFR] Updating blockers for test_ssa_schedule

### PR DESCRIPTION
**Pull request by jan-zmeskal**
Blocker

### CODE REVIEW

File: cfme/tests/cloud_infra_common/test_vm_instance_analysis.py [hunk 1]
```diff
@@ -592,8 +592,7 @@ def test_ssa_compliance(local_setup_provider, ssa_compliance_profile, ssa_vm,
 @pytest.mark.rhv3
 @pytest.mark.tier(2)
 @pytest.mark.long_running
-@pytest.mark.meta(blockers=[BZ(1578792, forced_streams=['5.8', '5.9'],
-    unblock=lambda provider: vm_system_type != 'redhat')])
+@pytest.mark.meta(blockers=[GH('ManageIQ/integration_tests:8157')])
```
Comment (sbulage) [inline cfme/tests/cloud_infra_common/test_vm_instance_analysis.py]: @jan-zmeskal I can see this PR 8157 has been closed. can you please cross-check it..?

**Comment by izapolsk** [comment]
PRT failed obtaining appliances. I've restarted PRT. lets wait for its result.

**Comment by sbulage** [comment]
@izapolsk it should not block all tests, only RHEV tests should be blocked right .?

File: cfme/tests/cloud_infra_common/test_vm_instance_analysis.py (hunk 1 above)
Comment (sbulage) [reply cfme/tests/cloud_infra_common/test_vm_instance_analysis.py]: Sorry for Typo :)

Comment (izapolsk) [reply cfme/tests/cloud_infra_common/test_vm_instance_analysis.py]: @sbulage, good point. @jan-zmeskal, as far as I understand you have this issue only against RHV, right ?

**Pull request merged**